strangerstudios/sitewide-sales
# Sale period settings refactor

### All Submissions:






### Changes proposed in this Pull Request:


Refactored blocks by restructuring code files. Incorporated new sale period setting block through modifying 'includes/blocks.php' and 'webpack.config'. Introduced a separate folder in 'sources' for new block files. Transferred code from 'blocks/block.js' and 'blocks/block.php' to individual files within the new block folder.

### How to test the changes in this Pull Request:

1. Make sure there is a sitewide sale active
2. insert group block
3. Go under advanced tab to look for sale period setting

### Other information:
The block is supposed to a add sale period setting under the advanced tab for container blocks like a group block. However, this setting doesn't show up at all. This block doesn't have a edit or save component. It also doesn't have any JSON settings, so I am unsure about what files to include for the block and what code to put into those files. 

* [ X] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ X] Have you successfully run tests with your changes locally?



### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
- Modifcations to blocks.php and webpack.config.js
- New src/sale-period-setting folder with block files
